src/NUnitEngine/nunit.engine.tests/Helpers/ProcessTester.cs

```diff
@@ -36,7 +37,8 @@ public static ProcessResult Run(ProcessStartInfo startInfo)
                 };
                 process.ErrorDataReceived += (sender, e) =>
                 {
-                    if (e.Data == null) return;
+                    if (e.Data == null)
+                        return;
```

Comment:
Another kind of thing where I find the rule overly pedantic, but at least it doesn't force adding parens.

Reply:
If it starts to annoy you we can disable rule SA1501, but until them leave it.